src/fhirdefs/FHIRDefinitions.ts

```diff
@@ -25,6 +25,11 @@ export class FHIRDefinitions extends BaseFHIRDefinitions implements Fishable {
     }
   }
 
+  // Expose the package.json files to support extracting the version when "latest" is used
+  allPackageJsons(): any[] {
+    return Array.from(this.packageJsons?.values() ?? []);
+  }
+
```

Comment:
I think it makes sense to include this in FPL, since that's where all the other `all*` functions are, like `allProfiles`, and the FPL base FHIRDefinitions defines the `packageJsons` list, so it isn't a SUSHI-specific concept. However, since we're trying to get this out in the next release, would you rather wait to do that?

Reply:
We talked about this offline, and we'll move this into FPL eventually, but we're okay with keeping it here until we're able to move into FPL. [CIMPL-1238](https://standardhealthrecord.atlassian.net/browse/CIMPL-1238) tracks that task.